sklearn/kernel_approximation.py

```diff
@@ -389,10 +389,10 @@ class Nystroem(BaseEstimator, TransformerMixin):
         the kernel; see the documentation for sklearn.metrics.pairwise.
         Ignored by other kernels.
 
-    degree : float, default=3
+    degree : float, default=None
```

Comment:
Changed what? Technically the default changed but not the behavior.
I can say "Changed default value from 3 to None", not sure if that's helpful, though.